.circleci/config.yml

```diff
@@ -2,6 +2,41 @@ version: 2.1
 
 jobs:
 
+  run-formatter:
+    docker:
+      - image: checkstyle/idea-docker:jdk11-idea2023.3.4
+
+    steps:
+      - checkout
+      - run:
+          name: Print versions
+          command: |
+            echo "Maven version:"
+            mvn --version
+            echo "Java version:"
+            java --version
+            echo "IDEA version:"
+            echo $IDEA_VERSION
+      - run:
+          name: Run code formatter
+          command: |
+            # Run the formatter
```

Comment:
```suggestion
            echo "Run the formatter"
```
might use or loose.

Reply:
@Anmol202005 , please just remove comment, we have it in name of run